.pre-commit-config.yaml

```diff
@@ -40,6 +40,15 @@ repos:
         files: ^packages\/sandbox
         types_or: [ts, tsx, javascript, jsx, json]
         pass_filenames: false
+      
+      - id: lint-statusLambda
+        name: Lint statusLambda
+        entry: npm
+        args: ["run", "--prefix=packages/statusLambda", "lint"]
+        language: system
+        files: ^packages\/statusLambda
+        types_or: [ts, tsx, javascript, jsx, json]
+        pass_filenames: false
```

Comment:
good catch. I forgot to add this in for gsul package - can you add it please

Reply:
lint-gsul has been added to pre-commit-config